### ISSUE TITLE
[a11y] Screen reader users should be able to navigate through the progress bar

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#280]
Issue:
- Terri wasn't able to read or announce the progress bar using the arrow keys with JAWS/NVDA. A screen reader user should have all the same content available to the person as a sighted person. I was able to arrow key to the text of the progress bar (VO+Safari), but it didn't announce steps, a user might not know what the items mean. 

Steps to recreate
- VO + Safari
- Navigate the page using the **UP/DOWN arrow keys** to read the text. After reading the last line of the intro, you'll notice that it reads "Step one of six: Contact" but no VO focus is showing. ([video](https://drive.google.com/open?id=1kKuwBpEoCHGNX2h-6ai-4S_4kdEXF25u))

Expected:
- Auditory cues should match visual cues in regards to current step and completed step. 
- Looking through documentation, I saw that you can put a hidden text label to announce completed/current. This "[navigation step indicator](https://design.gccollab.ca/component/progress-indicators/)" example was easier for me to use with VO, but here is [WCAGs as well "Approach 4: Using step-by-step indicator"](https://www.w3.org/WAI/tutorials/forms/multi-page/)"

**Comment by Jacklynn** [comment]
wooo! so exciting. 
* I noticed that it's still saying "step 1 of 7" I wonder if, because after step 6 voiceover says "new line." I wonder if it's picking up some code as another list item? @el-mapache

**Comment by Jacklynn** [comment]
Actually I noticed that the Details page is labeled as step 7, so maybe it's because we have the new human trafficking page?

![Screen Shot 2020-02-12 at 11.17.27 AM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/b4498f5a-e021-4a50-ba3a-d7b789987374)

**Comment by el-mapache** [comment]
@Jacklynn Yes, that is exactly correct. I'm not sure how avoidable it is because of the way the package we are using, `django-formtools`, handles its wizard step functionality. I can dig into it / consult with @LindsayYoung .

**Comment by Jacklynn** [comment]
This appears to be working now, announcing the correct number steps as visually shown, but the number isn't include in what's read allowed, is it possible to include the number as well?

**Comment by Jacklynn** [comment]
This was working Terri tested
